samples/test/export-model-tabular-classification.test.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const {assert} = require('chai');
+const {describe, it} = require('mocha');
+
+const cp = require('child_process');
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const cwd = path.join(__dirname, '..');
+
+const gcsDestinationOutputUriPrefix = 'gs://ucaip-samples-test-output';
+const modelId = '6036688272397172736';
+const project = process.env.CAIP_PROJECT_ID;
+const location = process.env.LOCATION;
```

Comment:
So silly question - how important is it that the LOCATION is configurable?  Is this one of those things we'll set once, and likely never change?  Trying to limit the number of custom env vars we use across the board for consistency.  If we need it, that's totally cool though.

Reply:
I hear you, and I'm pretty sure the answer is "not very" but there could be a Chesterton's Fence sort of thing here. I'm trying to make these Node samples conform kind of closely to the Python samples, which include the location.

Reply:
TIL about Chesterton's Fence - thank you!